checker/src/main/java/org/checkerframework/checker/signedness/SignednessVisitor.java

```diff
@@ -118,8 +118,7 @@ private boolean maskIgnoresMSB(Kind maskKind, LiteralTree shiftAmountLit, Litera
             // Check that the shiftAmount most significant bits of the mask were 1.
             return mask == (1 << shiftAmount) - 1;
         } else {
-            // This shouldn't be possible.
-            throw new RuntimeException("Invalid Masking Operation");
+            throw new RuntimeException("Invalid Masking Operation"); // unreachable
```

Comment:
For some of these the previous comment about `shouldn't happen` was more correct than `unreachable`.
If for whatever reason Java 12 adds a new maskKind, this branch will get executed.
I would use `unreachable` only if its really an imprecision in the analysis.

Reply:
Also, here and elsewhere, why use a `RuntimeException` or `IllegalArgumentException` instead of `BugInCF`?